src/System/Cron/Parser.hs

```diff
@@ -97,7 +99,7 @@ crontabEntry = A.skipSpace *> parser
 
 
 parseCronSchedule :: Text -> Either String CronSchedule
-parseCronSchedule = A.parseOnly cronSchedule
+parseCronSchedule = A.parseOnly cronSchedule . toLower
```

Comment:
Ah, sorry for the very slow response. The uppercase/lowercase stuff is handled by this.